src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,43 @@
+package duke;
+/**
+ * Deadlines is a task that have a deadline which is a string.
+ * @author Dominic Siew Zhen Yu
+ *
+ */
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task{
```

Comment:
You may want to change your class name to singular form (`deadline`) as plural forms are usually for collection of items.